test-case-reporting/static/test-case-list.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
```

Comment:
Does this not use the same test-case-list template all the other views use?

Reply:
There was no test-case-list template yet, only test-run-list and build-list templates.